### ISSUE TITLE
Build failure: floorp

### ISSUE THREAD

**Issue by hatch01** [NixOS/nixpkgs#320073]
### Steps To Reproduce

1. build *floorp*

### Build log

[log](https://gist.github.com/hatch01/64e9c6d40dd8bba6d862ec206d7de9a3)

### Notify maintainers

@christoph-heiss 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.3-zen1, NixOS, 24.11 (Vicuña), 24.11.20240612.57d6973`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/dydg48djlykksz8cxq0xjplyxpa9pvf4-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by christoph-heiss** [comment]
Hi,

Please elaborate - there does not seem to be any actual build failure here.
What's the _actual_ error? 
Are you sure your build machine has enough resources? A browser build is pretty demanding and can take even multiple hours on low-to-mid-end machines.

Can you please also post the _complete_ derivation log, as obtained using `nix log`?

**Comment by VincentBerthier** [comment]
I have the same problem (with floorp and floorp-unwrapped).
Here’s the error:
![image](https://github.com/NixOS/nixpkgs/assets/34085617/db186bf8-3d96-46e9-a0b1-4e772cf5f01b)

That feature was removed in February, so either update the qcms crate or downgrade the rust compiler version (1.76 could work).

**Comment by hatch01** [comment]
> Hi,
> 
> Please elaborate - there does not seem to be any actual build failure here. What's the _actual_ error? Are you sure your build machine has enough resources? A browser build is pretty demanding and can take even multiple hours on low-to-mid-end machines.
> 
> Can you please also post the _complete_ derivation log, as obtained using `nix log`?

is the log in the link not enough ?

**Comment by hatch01** [comment]
https://gist.github.com/hatch01/64e9c6d40dd8bba6d862ec206d7de9a3

**Comment by christoph-heiss** [comment]
Oh, I see, Github just truncated the file in the UI.
That's what you get from hosting projects on crappy, proprietary software .. 

Pointing out the exact error too from the get-go would have been helpful in any case.
Anyway - seems it broke with a recent rustc upgrade in nixpkgs.
This is already fixed with 11.14.1, as it updates the firefox base source. See #320114.

**Comment by hatch01** [comment]
Okay, I will host my opengist instance for the next time ;)
Thanks for your prompt response, waiting for your pr to be merged.

**Comment by cruthachail** [comment]
> Oh, I see, Github just truncated the file in the UI. That's what you get from hosting projects on crappy, proprietary software ..
> 
> Pointing out the exact error too from the get-go would have been helpful in any case. Anyway - seems it broke with a recent rustc upgrade in nixpkgs. This is already fixed with 11.14.1, as it updates the firefox base source. See #320114.

Any ETA on the PR merge?

**Comment by VincentBerthier** [comment]
Floorp is now in version 11.14.1 on unstable.
